Admin/AbstractAdmin.php

```diff
@@ -1042,7 +1042,7 @@ public function getActiveSubclassCode()
     /**
      * {@inheritdoc}
      */
-    public function getBatchActions()
+    final public function getBatchActions()
```

Comment:
This PR is for the next major release, so we can have a BC break. You can override any batch action with `configureBatchActions` as described in the docs.